osu.Game.Rulesets.Osu/UI/OsuPlayfield.cs

```diff
@@ -64,21 +67,54 @@ public override void Add(DrawableHitObject h)
 
             base.Add(h);
 
-            followPoints.AddFollowPoints((DrawableOsuHitObject)h);
+            DrawableOsuHitObject osuHitObject = (DrawableOsuHitObject)h;
```

Comment:
Getting the feeling this logic should be in its own class to keep `OsuPlayfield` from getting too polluted..